<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>socketDlg</class>
 <widget class="QDialog" name="socketDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通讯设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0,0,0,0">
   <item row="0" column="0" rowspan="2" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>状态：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>    端口：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_IP">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  连接IP：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_port">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_status">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>连接状态：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_listen">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>监听</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_connect">
          <property name="text">
           <string>connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit_sendData">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送内容（十六进制）：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit_beat">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="2">
    <widget class="QPushButton" name="pushButton_send">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit_receiveData">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>心跳：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_clearBeat">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清空心跳</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButton_clearReceive">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清空接受内容</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>接受内容（十六进制）：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="test">
     <property name="text">
      <string>test2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clearReceive</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_receiveData</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked()</signal>
   <receiver>socketDlg</receiver>
   <slot>on_pushButton_Connect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_Connect_clicked()</slot>
  <slot>slot_sendData()</slot>
  <slot>slot_sendWeldRet()</slot>
  <slot>test_new()</slot>
 </slots>
</ui>
